packages/app/src/cli/services/dev/extension/utilities.ts

```diff
@@ -58,3 +57,14 @@ export function getExtensionPointTargetSurface(extensionPointTarget: string) {
       return domain
   }
 }
+
+function isCheckoutRenderedCustomerAccountTarget(extensionPointTarget: string) {
+  const customerAccountCheckoutRenderedTargets = new Set([
+    'customer-account.order-status.cart-line-item.render-after',
+    'customer-account.order-status.cart-line-list.render-after',
+    'customer-account.order-status.customer-information.render-after',
+    'customer-account.order-status.block.render',
+  ])
```

Comment:
Nit: Since this is a constant you could extract it to avoid creating a new set on every function call.

Reply:
Moving these to a constant made the separate function redundant so the check is now being performed directly in `getExtensionPointTargetSurface`